rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -2,6 +2,15 @@ cmake_minimum_required(VERSION 3.5)
 
 project(rosidl_generator_cpp)
 
+# Default to C11
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
+endif()
```

Comment:
Why was this added? The package seems to only contain C++ code.

Reply:
we are compiling the test maybe include this inside `if(BUILD_TESTING) ... endif()` ?

Reply:
The tests have been built before this patch and are C++. So why would it be necessary now?